internal/kubernetes/operator/project/project.go

```diff
@@ -421,12 +421,12 @@ func buildIntegrations(intProvider store.IntegrationLister, projectID, targetNam
 			}
 		case newRelicIntegrationType:
 			integration.LicenseKeyRef = secretRef
-			secret.Data[secrets.PasswordField] = []byte(list.NewRelic.LicenseKey)
+			secret.Data[secrets.PasswordField] = []byte(list.GetLicenseKey())
 			// Secrets with write and read tokens
 			var writeToken, readToken string
 			if includeSecrets {
-				writeToken = list.NewRelic.WriteToken
-				readToken = list.NewRelic.ReadToken
+				writeToken = list.GetWriteToken()
```

Comment:
We should always use Getters to access fields

Reply:
@gssbzn We are still in a position to change all *strings to string in SDK at the cost of knowing if the field is optional or not.  Leaving this as topic for discussion.

Reply:
the sad face is because Getters are non idiomatic for go and seems like a code small introduced by the generator

Reply:
@gssbzn I'm adding this item to the agenda:

```
Optionality of the Strings in SDK
Optional Strings are represented as *string.
Result: We have to check for nullability and have nonstandard parsing behavior.
Q: Can we sacrifice optionality feature (less clarity) for strings (or even other types like integer, float), for the cost of less boilerplate code?
```

Reply:
we were using pointers on the manual SDK and we didn't implement getters, like I said the current implementation of having an exported field and a getter is a bit of a code smell for go having a public field but also a public getter that's not really a gtter but a "safe" getter while not being clear on it seems a bit of a weird design choice 

```go
type A struct {
  A *string
}

func (a *A) GetA() string {
  return &a.A
}

func main() {
  a := &A{}
  a.A
  a.GetA() 
}
```

Reply:
I understand. My question is - Can we leverage change for strings to be non pointer so then we do not need to have safe methods or use `pointer.Get(...)`. 

If we do that we going to lose information on what is an optional field.